cli/ceph/ceph.py

```diff
@@ -15,26 +12,17 @@ def __init__(self, nodes, base_cmd=""):
         self.base_cmd = f"{base_cmd} ceph" if base_cmd else "ceph"
         self.mgr = Mgr(nodes, self.base_cmd)
 
-    def version(self):
+    def version(self, **kw):
         """Get ceph version."""
         cmd = f"{self.base_cmd} version"
-        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
-        if isinstance(out, tuple):
-            return out[0].strip()
-        return out
+        return self.execute(cmd=cmd, **kw)
 
-    def status(self):
+    def status(self, **kw):
         """Get ceph status."""
         cmd = f"{self.base_cmd} status"
-        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
-        if isinstance(out, tuple):
-            return out[0].strip()
-        return out
+        return self.execute(cmd=cmd)
```

Comment:
accepting a kw arg is of no use here, as we are not passing to execute command

Reply:
Reverted this change. Not required now.